api/src/Microsoft.Azure.IIoT.OpcUa.Api/src/Core/Models/PendingAlarmsOptionsApiModel.cs

```diff
@@ -31,5 +31,11 @@ public class PendingAlarmsOptionsApiModel {
         [DataMember(Name = "snapshotInterval", Order = 2,
             EmitDefaultValue = false)]
         public int? SnapshotInterval { get; set; }
+
+        /// <summary>
+        /// Should we compress messages using GZip?
+        /// </summary>
+        [DataMember(Name = "compressedPayload", Order = 3)]
+        public bool CompressedPayload { get; set; } = false;
```

Comment:
Can we use the same format as the encoders ("UseReversibleEncoding", "UseAdvancedEncoding") for this... like "UseCompressedPayload" or "UseCompression"?

Reply:
No, this was an explicit ask from Alexander to use the same as in 3.0.